settings.gradle.kts

```diff
@@ -106,61 +53,32 @@ project(":clients:osv").name = "osv-client"
-project(":plugins:commands:upload-result-to-sw360").name = "upload-result-to-sw360-command"
-
-project(":plugins:package-curation-providers:api").name = "package-curation-provider-api"
-project(":plugins:package-curation-providers:clearly-defined").name = "clearly-defined-package-curation-provider"
-project(":plugins:package-curation-providers:file").name = "file-package-curation-provider"
-project(":plugins:package-curation-providers:ort-config").name = "ort-config-package-curation-provider"
-project(":plugins:package-curation-providers:sw360").name = "sw360-package-curation-provider"
-
-project(":plugins:package-managers:bower").name = "bower-package-manager"
-project(":plugins:package-managers:bundler").name = "bundler-package-manager"
-project(":plugins:package-managers:cargo").name = "cargo-package-manager"
-project(":plugins:package-managers:carthage").name = "carthage-package-manager"
-project(":plugins:package-managers:cocoapods").name = "cocoapods-package-manager"
-project(":plugins:package-managers:composer").name = "composer-package-manager"
-project(":plugins:package-managers:conan").name = "conan-package-manager"
-project(":plugins:package-managers:gradle").name = "gradle-package-manager"
-project(":plugins:package-managers:node").name = "node-package-manager"
-project(":plugins:package-managers:nuget").name = "nuget-package-manager"
-project(":plugins:package-managers:pub").name = "pub-package-manager"
-project(":plugins:package-managers:python").name = "python-package-manager"
-project(":plugins:package-managers:spdx").name = "spdx-package-manager"
-project(":plugins:package-managers:stack").name = "stack-package-manager"
-project(":plugins:package-managers:unmanaged").name = "unmanaged-package-manager"
-
-project(":plugins:reporters:asciidoc").name = "asciidoc-reporter"
-project(":plugins:reporters:ctrlx").name = "ctrlx-reporter"
-project(":plugins:reporters:cyclonedx").name = "cyclonedx-reporter"
-project(":plugins:reporters:evaluated-model").name = "evaluated-model-reporter"
-project(":plugins:reporters:fossid").name = "fossid-reporter"
-project(":plugins:reporters:freemarker").name = "freemarker-reporter"
-project(":plugins:reporters:gitlab").name = "gitlab-reporter"
-project(":plugins:reporters:opossum").name = "opossum-reporter"
-project(":plugins:reporters:spdx").name = "spdx-reporter"
-project(":plugins:reporters:static-html").name = "static-html-reporter"
-project(":plugins:reporters:web-app").name = "web-app-reporter"
-
 project(":utils:common").name = "common-utils"
 project(":utils:ort").name = "ort-utils"
 project(":utils:scripting").name = "scripting-utils"
 project(":utils:spdx").name = "spdx-utils"
 project(":utils:test").name = "test-utils"
 
+file("plugins").walk().maxDepth(3).filter {
+    it.isFile && it.name == "build.gradle.kts"
+}.mapTo(mutableListOf()) {
+    it.parentFile.toRelativeString(rootDir).replace(File.separatorChar, ':')
+}.forEach {
+    include(":$it")
```

Comment:
It wasn't super obvious to me what these two `it`s in line 65 and 67 refer to.
How about fixing this by using some variable, like e.g:

```
val moduleNames = file("plugins").walk().maxDepth(3).filter {
    it.isFile && it.name == "build.gradle.kts"
}.mapTo(mutableListOf()) {
    ":${it.parentFile.toRelativeString(rootDir).replace(File.separatorChar, ':')}" 
}

moduleNames.forEach { moduleName ->
     include(moduleName)
     ...
}
```